src/components/auth/ForgotPassword.js

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from "react"
+import { authPass } from "../../state/auth/authActionCreator"
+import { connect, useDispatch } from "react-redux"
+import { validateEmail } from "./utils"
+import { Link } from "react-router-dom"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['Link' is defined but never used.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5684995)